src/test/scala/no/ndla/audioapi/controller/AudioControllerTest.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Part of NDLA audio_api.
+ * Copyright (C) 2016 NDLA
+ *
+ * See LICENSE
+ *
+ */
+
+package no.ndla.audioapi.controller
+
+import no.ndla.audioapi.model.api._
+import no.ndla.audioapi.{AudioSwagger, TestEnvironment, UnitSuite}
+import org.mockito.Matchers._
+import org.mockito.Mockito._
+import org.scalatra.servlet.FileItem
+import org.scalatra.test.Uploadable
+import org.scalatra.test.scalatest.ScalatraSuite
+
+import scala.util.{Failure, Success}
+
+class AudioControllerTest extends UnitSuite with ScalatraSuite with TestEnvironment {
```

Comment:
Hmm...fint om denne kunne hatt samme navn som selve controlleren + Test. Er vel kanskje den faktiske kontrolleren som bør hete `AudioController` i stedet for `AudioApiController`